javascripts/discourse/templates/components/topic-footer-buttons.hbs

```diff
@@ -0,0 +1,70 @@
+<div class='topic-footer-main-buttons'>
+  {{#if showAdminButton}}
+  {{topic-admin-menu-button
+      topic=topic
+      openUpwards="true"
+      toggleMultiSelect=toggleMultiSelect
+      hideMultiSelect=hideMultiSelect
+      deleteTopic=deleteTopic
+      recoverTopic=recoverTopic
+      toggleClosed=toggleClosed
+      toggleArchived=toggleArchived
+      toggleVisibility=toggleVisibility
+      showTopicStatusUpdate=showTopicStatusUpdate
+      showFeatureTopic=showFeatureTopic
+      showChangeTimestamp=showChangeTimestamp
+      resetBumpDate=resetBumpDate
+      convertToPublicTopic=convertToPublicTopic
+      convertToPrivateMessage=convertToPrivateMessage}}
+  {{/if}}
+
+  {{#if site.mobileView}}
+  {{topic-footer-mobile-dropdown topic=topic content=dropdownButtons}}
+  {{/if}}
+
+  {{#each inlineButtons as |button|}}
+  {{d-button
+      id=(concat "topic-footer-button-" button.id)
+      class=(concat "btn-default topic-footer-button " button.classNames)
+      action=button.action
+      icon=button.icon
+      translatedLabel=button.label
+      translatedTitle=button.title
+      disabled=button.disabled}}
+  {{/each}}
+
+  {{plugin-outlet name="topic-footer-main-buttons-before-create"
+                  args=(hash topic=topic)
+                  tagName=""
+                  connectorTagName="span"}}
+
+  {{#if topic.details.can_create_post}}
+  {{d-button class="btn-primary create"
+               icon="reply"
+               action=replyToPost
+               label="topic.reply.title"
+               title="topic.reply.help"}}
+  {{else}}
+  {{d-button class="btn-primary create disabled"
+               icon="reply"
+               action=replyToPost
+               label="topic.reply.title"
+               title="discourse_debtcollective_collectives.no_permission"}}
+  {{/if}}
+
+  {{plugin-outlet name="after-topic-footer-main-buttons"
+                  args=(hash topic=topic)
+                  tagName=""
+                  connectorTagName="span"}}
+
+  {{pinned-button pinned=topic.pinned topic=topic}}
+</div>
+
+{{#if showNotificationsButton}}
+{{topic-notifications-button notificationLevel=topic.details.notification_level topic=topic}}
```

Comment:
What happen with your formatter? something like this one I think we already installed within your machine:
https://marketplace.visualstudio.com/items?itemName=EmberTooling.prettier-for-handlebars-vscode